### PR TITLE
Created schema

### PR DESCRIPTION
**Pull request by vflorin1995**
### Changes:

- Created schema from the picture

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    id int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    type varchar,
+    name varchar
+);
+
+CREATE TABLE medical_histories_treatment (
+    medical_histories_id int REFERENCES medical_histories (id),
+    treatment_id int REFERENCES treatments (id),
+    date timestamp,
+    PRIMARY KEY(medical_histories_id, treatment_id, date)
```
Comment (omar-muhamad) [inline schema_based_on_diagram.sql]: - [ ] You don't need a `date` and `id` for a join table that addresses many to many relationship, So please remove them from the table.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+CREATE TABLE invoice_items (
+    id int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    unit_price decimal,
+    quantity int,
+    total_price decimal,
+    invoice_id int REFERENCES invoices (id),
+    treatment_id int REFERENCES treatments (id)
+);
+
```
Comment (omar-muhamad) [inline schema_based_on_diagram.sql:42]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.), Please make sure that all FOREIGN KEYS have indexes created.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @vflorin1995 and @ndikumanaisaie,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omarsalem7 — APPROVED** [review]
Hi @vflorin1995 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Index added for all foreign keys
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**